tests/dummy-live-app/package.json

```diff
@@ -25,11 +25,11 @@
     ]
   },
   "devDependencies": {
-    "@types/jest": "^27.0.1",
```

Comment:
i'm seeing a version bump here instead of pinning

Reply:
see comment above

Reply:
ok thank you